mongoexport/json.go

```diff
@@ -24,20 +22,20 @@ type JSONExportOutput struct {
 	ArrayOutput bool
 	// Pretty when set to true indicates that the output will be written in pretty mode.
 	PrettyOutput bool
-	Encoder      *json.Encoder
 	Out          io.Writer
 	NumExported  int64
+	JSONFormat   string
```

Comment:
It might be worth defining an enum representing the different possible cases and storing that rather than just storing a raw string here.

Reply:
done in new commit